src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.tools.FigureSupplier;
+
+public class Main {
+    private static FigureSupplier figureCreater = new FigureSupplier();
+
+    public static void main(String[] args) {
+        final int sizeLists = 6;
+        Figure[] figureLists = new Figure[sizeLists];

```

Comment:
```suggestion
        final int arraySize = 6;
        Figure[] figures = new Figure[arraySize];
```
please use proper naming

Reply:
not fixed